.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
+  branch = feature/openmp_variant_eckit_fckit_atlas
```

Comment:
are the url/branch in .gitmodules intentionally changed here or was this only done for testing?

Reply:
I see you passed the "careful review" test ;-) Everyone asks the question the first time, rightfully, because this is not something that should get merged. It's needed - like for the ufs-weather-model - so that the CI tests check out the updated spack code and test with it. Once the spack PR is merged (I will wait for approvals and the CI tests), then `.gitmodules` gets reverted and the spack submodule pointer will be pointed back to the (new) head of jcsda_emc_spack_stack branch in the noaa-emc spack fork. Only after that this PR can be merged.

Reply:
@climbfuji haha, glad i was able to catch a red herring here. what was the old saying? teach a man to fish and he might leave extraneous comments on github PR reviews for life? 

but that now makes sense and i appreciate the description. noted for future PRs.